cmd/lukso/download.go

```diff
@@ -204,6 +209,14 @@ func downloadPrysmConfig(ctx *cli.Context) (err error) {
 	return
 }
 
+func downloadGethConfig(ctx *cli.Context) (err error) {
+	log.WithField("dependencyTag", prysmTag).Info("Downloading Geth Config")
```

Comment:
Those are configs, so it doesn't really matter that much, but just for the sake of avoiding any confusions I deleted these log fields, now we simply download a config - without a version, because there isn't any attached to one.